app/index.html

```diff
@@ -210,17 +241,28 @@ <h3>Skills</h3>
     <footer class="u-margin primary-font primary-text-color">
         <section id="form-container">
             <div id="bg-footer"></div>
-            <h2 id="form-text">I&apos;m always interested in hearing about new projects, so if you&apos;d like to chat please get in touch.</h2>
-            <form action="https://formspree.io/f/mvolzpve" method="post">
-                <input class="text-fields" type="text" id="name" name="user_name" placeholder="Full name" required maxlength="30">
-                <input class="text-fields" type="email" id="email" name="email" placeholder="Email address" required>
-                <textarea class="text-fields primary-font" name="user_message" id="message" placeholder="Enter text here..." required maxlength="500"></textarea>
-                <div>
-                    <button class="button work-item-button primary-font" type="submit">Get in touch</button>
-                </div>
-            </form>
+            <div id="bg-footer-d-r" class="desktop-footer-bg"></div>
```

Comment:
- [ ] Please kindly correct the font styles in "I'm always interested...".  
- [ ] The `textarea` of the form should have a green border, like in the design.  Let's change it.
- [ ] As the requirements state, you should use only one form field for the name, not 2.  Kindly delete one and use it for the full name.

![image](https://user-images.githubusercontent.com/57421823/149352022-63578975-4094-48c2-aaa3-4d6e446417ca.png)

That way, your project will look like the template 💪

Reply:
**[Corrected]**

1. font styles in "I'm always interested...".
2. Removed two input name fields for desktop version.


`textarea` supposed to change it's border when active (user clicked on it to type the message); what do you think?

Reply:
All issues were addressed 🔥 Your textarea is working as expected 😉